rfc-001-announcement-channel-protocol.md

```diff
@@ -0,0 +1,28 @@
+# What content is applicable for the announcements channel
+
+## Summary
+
+What types of messages are relevant to the newly renamed announcements channel?
+
+## Problem
+
+Off the back of the Deep Work retro hosted by Chris, we had common consensus that #general too noisy.
+
+To get round that we've created a new channel, #common-room for any chatter, and renamed general to announcements.
+
+There is a lot of ambiguity about what content is suitable for the new channel.
```

Comment:
Might be worth saying announcements instead of the new channel given the same phrasing is used above for common room